sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TransmitFromStorageHandler.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Timers;
+using OpenTelemetry;
+using OpenTelemetry.Extensions.PersistentStorage.Abstractions;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal class TransmitFromStorageHandler : IDisposable
+    {
+        private readonly ApplicationInsightsRestClient _applicationInsightsRestClient;
+        private readonly PersistentBlobProvider _blobProvider;
+        private readonly TransmissionStateManager _transmissionStateManager;
+        private readonly System.Timers.Timer _transmitFromStorageTimer;
+        private bool _disposed;
+
+        internal TransmitFromStorageHandler(ApplicationInsightsRestClient applicationInsightsRestClient, PersistentBlobProvider blobProvider, TransmissionStateManager transmissionStateManager)
+        {
+            _applicationInsightsRestClient = applicationInsightsRestClient;
+            _blobProvider = blobProvider;
+            _transmissionStateManager = transmissionStateManager;
+            _transmitFromStorageTimer = new System.Timers.Timer();
+            _transmitFromStorageTimer.Elapsed += TransmitFromStorage;
+            _transmitFromStorageTimer.AutoReset = true;
+            _transmitFromStorageTimer.Interval = 120000;
```

Comment:
Is there a reason to select this interval?

Reply:
This is arbitrary. Open for any better suggestions.